lib/redirect.js

```diff
@@ -0,0 +1,146 @@
+'use strict'
+
+var url = require('url')
+var isUrl = /^https?:/
+
+function Redirect (request) {
+  this.request = request
+  this.followRedirect = true
+  this.followRedirects = true
+  this.followAllRedirects = false
+  this.allowRedirect = function () {return true}
```

Comment:
I'm adding all of the object fields in the ctor, just like the modules are required on top, so that you know what's used and what's available. It's like a convention or something. Plus this module will definitely get refactored a bit more, at some point.